### PR TITLE
[Homepage] Add new hero graphic

### PR DESCRIPTION
**Pull request by daniel-panhead**
image

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            fill="currentColor"
+            viewBox="0 0 16 16"
+          >
+            <path d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1zm13 2.383-4.708 2.825L15 11.105zm-.034 6.876-5.64-3.471L8 9.583l-1.326-.795-5.64 3.47A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.741M1 11.105l4.708-2.897L1 5.383z" />
+          </svg>
+        </a>
+        <a href="https://instagram.com/ubcbiomod">
```
Comment (CarsonKiibi) [inline src/components/Footer.tsx]: imo ``` target="_blank" ``` for these links but maybe thats just personal preference

File: src/components/main-page/Hero.tsx [hunk 2]
```diff
@@ -8,18 +9,23 @@ import dots from "@/assets/dots.png";
 export default function Hero() {
   return (
     <div className="flex flex-col gap-4 lg:gap-10 px-4 lg:px-20">
-      <div className="flex flex-col lg:flex-row-reverse items-center justify-center">
+      <Image
+        className="hidden lg:block absolute right-0 top-14 w-full"
+        src={cellDesktopImage}
```
Comment (CarsonKiibi) [inline src/components/main-page/Hero.tsx]: Img looks a bit fuzzy on 24 inch monitor, not super bad but noticeable

**Review by CarsonKiibi — COMMENTED** [review]
Looks great overall. Footer is super useful and looks great.

Mobile view cellImage takes up nearly half of mobile view, I feel like it could go off the top of the view or something and only be half showing to allow for more space for other content.  But that could also make viewer think they are missing out on something on top haha. Not necessary to change but thought id mention it

**Comment by daniel-panhead** [comment]
> Looks great overall. Footer is super useful and looks great.
> 
> Mobile view cellImage takes up nearly half of mobile view, I feel like it could go off the top of the view or something and only be half showing to allow for more space for other content. But that could also make viewer think they are missing out on something on top haha. Not necessary to change but thought id mention it

hmm my thoughts on mobile is that it helps form like a complete fullscreen hero. But I'm not married to it

File: src/components/main-page/Hero.tsx (hunk 2 above)
Comment (daniel-panhead) [reply src/components/main-page/Hero.tsx]: :sob:

**Comment by daniel-panhead** [comment]
Comments addressed - desktop image is svg now for infinite scaling + fixed footer links

**Review by IcedTea2K — APPROVED** [review]

**Pull request merged**